apps/web/src/components/borrow/table/TableInput.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  BaseInput,
+  Button,
+  ColumnData,
+  ColumnLayout,
+  StackLayout,
+  Table,
+  Text,
+} from '@bloxifi/ui'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { ReservesData } from '@/containers/WalletContainer'
+
+interface Props {
+  handleInputChange: (value: string) => void
+  amount: string
```

Comment:
we should probably type this to number and add `<input type="number"`

Reply:
done